.github/CODE_STANDARDS.md

```diff
@@ -340,25 +357,13 @@ Adhering to documentation code standards is integral for maintaining a healthy,
     - [1.1 In GO applications or libraries, we follow the official guidelines](#11-in-go-applications-or-libraries-we-follow-the-official-guidelines)
```

Comment:
don't we have whole code standards twice here in this file?

Reply:
you are right - I don't know why - removed redundant part